grpc_interceptor/user.go

```diff
@@ -0,0 +1,41 @@
+package grpc_interceptor
```

Comment:
should we move this package to be in the `api` folder since it is being used there and it has  grpc request/response context?

Reply:
Yes that could be done

Reply:
@StewartJingga after checking again through the code, this is being used in `cmd/server.go` but I think this still could be moved to the `api` package